VAMobile/e2e/tests/HomeScreen.e2e.ts

```diff
@@ -4,9 +4,9 @@ import { CommonE2eIdConstants, loginToDemoMode, backButton } from './utils'
 export const HomeE2eIdConstants = {
 	GREETING_ID: 'greeting-text',
 	PAYMENTS_BTN_ID: 'Payments',
-	LOCATION_FINDER_ROW_ID: 'Find a V-A Location',
-	CONTACT_VA_ROW_ID: 'Contact V-A',
-	COVID_ROW_ID: 'V﻿A COVID-19 updates',
+	LOCATION_FINDER_ROW_ID: 'Find a  V-A  location',
```

Comment:
@rbontrager I don't know if we want to deal with this this way, but I'd guess the React Native code would need to be updated otherwise